server/application/application.go

```diff
@@ -115,7 +115,10 @@ func NewServer(
 	if appBroadcaster == nil {
 		appBroadcaster = &broadcasterHandler{}
 	}
-	appInformer.AddEventHandler(appBroadcaster)
+	_, err := appInformer.AddEventHandler(appBroadcaster)
+	if err != nil {
+		return nil, nil
+	}
```

Comment:
I think this is a subtle change in behavior. To keep with previous behavior, we should just log.debug and then continue.

Reply:
done